src/Light/2band.jl

```diff
@@ -1,7 +1,7 @@
 @kernel function update_TwoBandPhotosyntheticallyActiveRadiation!(PAR, grid, P, surface_PAR, t, PAR_model) 
     i, j = @index(Global, NTuple)
 
-    x, y = xnode(i, grid, Center()), ynode(j, grid, Center())
+    x, y = @inbounds xnodes(grid, Center(), Center(), Center())[i], ynodes(grid, Center(), Center(), Center())[j]
```

Comment:
not sure if `k=1` is what we want here, but how about:
```suggestion
    x = xnode(i, j, 1, grid, Center(), Center(), Center())
    y = ynode(i, j, 1, grid, Center(), Center(), Center())
```

Reply:
Note that for general curvilinear grids, the nodes depend on both `i, j`. So you probably want to use @navidcy's suggestion.

All current grids (and for the forseeable future) are independent of `k` so it's ok to take `k=1`. This makes a "thin spherical shell" approximation for the ocean.